Source/WebCore/bindings/js/JSElementInternalsCustom.cpp

```diff
@@ -38,6 +38,37 @@
 namespace WebCore {
 using namespace JSC;
 
+JSValue JSElementInternals::setFormValue(JSGlobalObject& lexicalGlobalObject, CallFrame& callFrame)
```

Comment:
It's unfortunate that we have to use custom bindings code here.

Reply:
It indeed is, yet unfortunately it's the best way to fix the issue without revamping a lot of bindings-related code. I will make sure to explain why in ChangeLog.